cloudaux/aws/iam.py

```diff
@@ -402,3 +417,87 @@ def get_account_authorization_details(filter=None, client=None, **kwargs):
         return _get_account_authorization_local_managed_policies_details(client=client, **kwargs)
     elif filter == 'AWSManagedPolicy':
         return _get_account_authorization_aws_managed_policies_details(client=client, **kwargs)
+
+
+@paginated('Users')
+@rate_limited()
+def _get_users_for_group(client, **kwargs):
+    """Fetch the paginated users attached to the group."""
+    return client.get_group(**kwargs)
+
+
+@sts_conn('iam', service_type='client')
+@rate_limited()
+def get_group(group_name, users=True, client=None, **kwargs):
+    """Get's the IAM Group details.
+
+    :param group_name:
+    :param users: Optional -- will return the IAM users that the group is attached to if desired (paginated).
+    :param client:
+    :param kwargs:
+    :return:
+    """
+    # First, make the initial call to get the details for the group:
+    result = client.get_group(GroupName=group_name, **kwargs)
+
+    # If we care about the user details, then fetch them:
+    if users:
```

Comment:
Since the boto3 IAM client's `get_group()` will returns a paginated response with a user list, maybe worth checking the `IsTruncated` response flag to see if we got the full list and use that rather than going to `_get_users_for_group` as a matter of course.

Reply:
Fixed the logic.